adapters/handlers/rest/handlers_schema.go

```diff
@@ -274,7 +274,7 @@ func (s *schemaHandlers) deleteTenants(params schema.TenantsDeleteParams,
 func (s *schemaHandlers) getTenants(params schema.TenantsGetParams,
 	principal *models.Principal,
 ) middleware.Responder {
-	tenants, err := s.manager.GetConsistentTenants(params.HTTPRequest.Context(), principal, params.ClassName, *params.Consistency, nil)
+	tenants, err := s.manager.GetConsistentTenants(params.HTTPRequest.Context(), principal, params.ClassName, *params.Consistency, nil, nil, nil)
```

Comment:
Do we really want to update the `schema.TenantsGetParams` given after and limit are really not needed in this function; I needed to bring them in since the function uses `h.metaWriter.QueryTenants`. 🫤